tests/test_pipelines_fill_mask.py

```diff
@@ -340,3 +338,27 @@ def fill_mask_with_duplicate_targets_and_top_k(self, model, tokenizer):
         # The target list contains duplicates, so we can't output more
         # than them
         self.assertEqual(len(outputs), 3)
+
+    def fill_mask_with_multiple_masks(self, model, tokenizer):
```

Comment:
Can we perhaps add a test for Perceiver (similar to the image classification models)?

Reply:
I think this PR is pretty orthogonal to Perceiver.

We could add a slow test for sure, but it doesn't have to be Perceiver specific.

In fact, I'll add something on the random model (it just needs to be consistent, actual values are less important)